pkg/scheduler/internal/cache/node_tree.go

```diff
@@ -110,14 +110,14 @@ func (nt *NodeTree) removeNode(n *v1.Node) error {
 				if len(na.nodes) == 0 {
 					nt.removeZone(zone)
 				}
-				klog.V(5).Infof("Removed node %v in group %v from NodeTree", n.Name, zone)
+				klog.V(2).Infof("Removed node %q in group %q from NodeTree", n.Name, zone)
 				nt.numNodes--
 				return nil
 			}
 		}
 	}
-	klog.Errorf("Node %v in group %v was not found", n.Name, zone)
-	return fmt.Errorf("node %v in group %v was not found", n.Name, zone)
+	klog.Errorf("Node %q in group %q was not found", n.Name, zone)
```

Comment:
Is the Warning level sufficient? Or this line of error logs can be deleted because the error has been returned.

Reply:
I guess it doesn't hurt to log an error here for debugging purposes, otherwise we have to rely on the caller to log a proper error message somewhere.

Reply:
I rather prefer that an error is only logged once. Otherwise it's hard to tell whether an error is unique or not. Could you please check if errors returned from this function are properly logged at some point? If so, this line could be removed.

Reply:
ha, the error is just ignored... I am going to fix it.

Reply:
I think it's a pattern to log an error message in the source of error. Otherwise it's hard to enforce all callers to log a proper error message. I will leave the error message here.

Reply:
sg